src/amber/cli/templates/app/config/environments/production.yml.ecr

```diff
@@ -1,14 +1,12 @@
 secret_key_base: <%= SecureRandom.urlsafe_base64(32) %>
-port: <%= (ENV["PORT"] ||= "80").to_i %>
+port: 8080
 name: <%= @name %>
-log: "::Logger.new(STDOUT)"
```

Comment:
How do we configure loggers per environment this way?

Reply:
Honestly defining a Logger should not be part of a setting. I am considering to move the logger to `Amber.logger`

Reply:
Would it be possible to completely configure the logger through the pipeline rather than here?

`plug Amber::Pipe::FastLogger.new(stdout: Amber.env.development?)`